src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentsPageV2.jsx

```diff
@@ -49,6 +69,11 @@ function AppointmentsPageV2({
   startNewExpressCareFlow,
   showHomePageRefresh,
 }) {
+  const location = useLocation();
+  const [dropdownValue, setDropdownValue] = useState(
+    getDropdownValueFromLocation(location.pathname),
+  );
```

Comment:
Dropdown list was defaulting to "Upcoming" if navigating directly to the "Requested" page.  This stores and sets the dropdown value on load based on the pathname.